src/poliastro/twobody/orbit.py

```diff
@@ -1241,27 +1242,6 @@ def propagate_to_anomaly(self, value):
             plane=self.plane,
         )
 
-    def _sample_closed(self, values, min_anomaly, max_anomaly):
-        min_anomaly = (
-            min_anomaly.to(u.rad).value
-            if min_anomaly is not None
-            else self.nu.to(u.rad).value
-        )
-        max_anomaly = (
-            max_anomaly.to(u.rad).value
-            if max_anomaly is not None
-            else self.nu.to(u.rad).value + 2 * np.pi
-        )
-        limits = [min_anomaly, max_anomaly] * u.rad
-
-        # First sample eccentric anomaly, then transform into true anomaly
-        # to minimize error in the apocenter, see
-        # http://www.dtic.mil/dtic/tr/fulltext/u2/a605040.pdf
-        # Start from pericenter
-        E_values = np.linspace(*limits, values)
```

Comment:
Look! We were making a `linspace` of eccentric anomalies... using true anomalies. This was wrong.